bin/generate-readme

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+transcribe() {
+  local -r project="$(node --print 'require("./package.json").repository.url.replace(/[.]git$/, "").split("/").slice(-2).join("/")')"
```

Comment:
The `repository` field in a `package.json` doesn't have to be an object with a `url` property. It [can be a string representation](https://docs.npmjs.com/files/package.json#repository) with a variety of formats. Do we want to support these formats? If not, we should mention that in the README.